### ISSUE TITLE
Show Preview Images in Map

### ISSUE THREAD

**Issue by Urmel** [nemethviktor/GeoTagNinja#61]
When the position of selected images is shown on the map, I would like to also see (if I want that) preview images at these location.

The only use case I know that I would have liked it to use it for is when reviewing a GPX track sync result. Apart from that, it might just look cool.... :)

**Comment by nemethviktor** [comment]
I think this might be one of those not-quite-possible/sensible things. 

I kinda see a few problems. One is that we have 1 picturebox atm, but I'm assuming you'd actually like to see the images _on the map_, not the picturebox.

I have, from my oooold times literal thousands of photos from some city trip (we can at this point argue that I should delete them, they're archived but that's not to the point.). I just geotagged them all with "Amsterdam" because it's a trip I made well before smart phones became a thing and I don't know AMS well enough/not bothered/etc to do a detailed breakdown. Basically a _lot_ of pics have 1 set of coords. While this is unlikely to be an issue with TrackSync, I think we do have to account for the idea that some users have lots of pics either per coordinate or just in general. It would be difficult to display these anywhere.

The other thing is that we currently inject the coordinates into the html code and have that be loaded in. My javascript skills are "level 0" to quote kung-fu-panda but I think it should be possible to add in an extra line or two to have it also display an image in say 50x50px, tho' this would need to be worked out somewhat subject to monitor size/pixel count/something as a given img size looks good on one screen but not the other. Alas, back to the original query, where do we put possibly hundreds of pictures? :)
